proto/lbm/wasm/v1/genesis.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
```

Comment:
This `syntax` indicate this proto will define as proto3 version of the proto buffers langauge.
https://developers.google.com/protocol-buffers/docs/proto3

Reply:
Sorry, I ask about the `genesis.proto` file.

Reply:
Oh, sorry. This `genesis.proto` file extend cosmwasm's `genesis.proto` file. And lbm-sdk don't use cosmwasm's `genesis.proto` file and use lbm-sdk's `genesis.proto`. Because the param of wasm genesis is difference with cosmwasm. And I'll have a plan to add the blacklist data in this genesisState message at next PR.